prometeo/mem/ast_analyzer.py

```diff
@@ -22,13 +22,15 @@
     'global@pmt_gemm_tn': [], \
     'global@pmt_gemm_nt': [], \
     'global@pmt_trmm_rlnn': [], \
+    'global@pmt_syrk_ln': [], \
     'global@pmt_gead': [], \
     'global@pmt_potrf': [], \
```

Comment:
`potrf_l`

Reply:
I thought it might make sense to try to have API's of different level of abstraction and expose a simplified `potrf`, where lower is implicit, and a more verbose and versatile one where one could also pass the `lower/upper` and `normal/transposed` arguments. I still haven't found out a systematic way of defining these abstraction levels of the API, so any recommendation is more than welcome :)